.mocharc.dom.json

```diff
@@ -0,0 +1,13 @@
+{
+	"require": [
+		"ts-node/register",
+		"./client/lib/tests/registerJsdom.ts"
+	],
+	"reporter": "spec",
+	"ui": "bdd",
+	"extension": ["ts", "tsx"],
+	"spec": [
+		"client/**/*.spec.ts",
+		"client/**/*.spec.tsx"
```

Comment:
this glob finds tests other than DOM unit tests (it finds `client/lib/minimongo/bson.spec.ts` for example).. idk how bad is that